etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -37,7 +37,9 @@ stackhpc_overcloud_dib_elements:
 stackhpc_overcloud_dib_env_vars:
   DIB_BLOCK_DEVICE_CONFIG: "{{ stackhpc_overcloud_dib_block_device_config_uefi_lvm }}"
   DIB_BOOTLOADER_DEFAULT_CMDLINE: "nofb nomodeset gfxpayload=text net.ifnames=1 rd.auto"
-  DIB_CLOUD_INIT_DATASOURCES: "ConfigDrive"
+  DIB_GRUB_TIMEOUT: "5"
+  DIB_GRUB_TIMEOUT_STYLE: "countdown"
+  DIB_CLOUD_INIT_DATASOURCES: "OpenStack, ConfigDrive"
```

Comment:
Default from Kayobe is ConfigDrive (https://github.com/openstack/kayobe/blob/3c178846a4e40bfc1c5f6f3bed2f5f70bf226cf0/ansible/inventory/group_vars/all/overcloud-dib#L59) - add a reno please

Reply:
Actually we did set that here, but still deserves a reno